mappings/net/minecraft/enchantment/Enchantment.mapping

```diff
@@ -15,44 +15,61 @@ CLASS net/minecraft/unmapped/C_jxtrubuh net/minecraft/enchantment/Enchantment
 		ARG 0 id
 	METHOD m_fecfwekl getRarity ()Lnet/minecraft/unmapped/C_jxtrubuh$C_dnkiztld;
 	METHOD m_glfcpyqa getMaxLevel ()I
+		COMMENT Sets the maximum number of levels the enchantment has. Default: 1.
 	METHOD m_igxrorrr getMinLevel ()I
+		COMMENT Sets the minimum number of levels the enchantment has. Default: 1.
 	METHOD m_jtamscxb canCombine (Lnet/minecraft/unmapped/C_jxtrubuh;)Z
 		COMMENT {@return whether this enchantment can exist on an item stack with the
 		COMMENT {@code other} enchantment and the {@code other} enchantment can exist
 		COMMENT with this enchantment}
 		ARG 1 other
 	METHOD m_khmhdqpw isAcceptableItem (Lnet/minecraft/unmapped/C_sddaxwyk;)Z
+		COMMENT {@return the type of item ({@code stack}) that can have this applied to it}
 		ARG 1 stack
 	METHOD m_kzbinilx getAttackDamage (ILnet/minecraft/unmapped/C_amnulxyk;)F
+		COMMENT {@return the amount of extra damage to apply to the entity group specified with {@code group}.}
 		ARG 1 level
 		ARG 2 group
 	METHOD m_lkqpuckl isTreasure ()Z
+		COMMENT {@return whether this enchantment will appear in the enchantment table and loot chests or loot chests only. E.G Mending}
```

Comment:
You should add that villagers can also sell these enchantments, but they sell it at twice the price they would with a normal enchantment

Reply:
Fixed